src/Umbraco.Core/Services/ContentTypeServiceBaseOfTRepositoryTItemTService.cs

```diff
@@ -304,7 +304,8 @@ public IEnumerable<TItem> GetAll(IEnumerable<Guid>? ids)
                 return Enumerable.Empty<TItem>();
             }
 
-            using (var scope = ScopeProvider.CreateScope(autoComplete: true))
+            using (ICoreScope scope = ScopeProvider.CreateCoreScope())
```

Comment:
This change removed the `autoComplete: true`, which caused the whole database transaction to silently rollback. Because there's no exception thrown, the repository cache was updated (if it's also doing an update in the same scope), even through it wasn't persisted to the database 💥

This caused the following `Cannot save a non-current version` error while trying to save/publish content for a second time, but only if it contained a Block List editor with values. It only happed in this specific case, because the editor calls this method while creating/updating property values, so the first save/publish wouldn't even be persisted to the database, but it was added to the repository cache and returned to the back-office. The second save/publish then couldn't find the version in the database and showed this error:

![Cannot save a non-current version](https://user-images.githubusercontent.com/1051287/166656257-7f0ece91-b6ed-48f0-bc57-025f7d2b757e.png)

This is fixed in https://github.com/umbraco/Umbraco-CMS/commit/6752f4b2c676c0225cb653c58b1fccb431f7b703 and I've carefully reviewed all other changes in this PR, so can confirm this is the only one👍🏻